### PR TITLE
docs(csharp): events convention based naming fix for csharp

### PR DESCRIPTION
**Pull request by avodovnik**
Updating some event names to match the expected convention in dotnet land.

### CODE REVIEW

**Review by pavelfeldman — COMMENTED** [review]
We should also swap the defaults here and lowercase event names in JavaScript and Python, because lowercasing is lossy. Then we again have no aliases.

**Comment by avodovnik** [comment]
> We should also swap the defaults here and lowercase event names in JavaScript and Python, because lowercasing is lossy. Then we again have no aliases.

Did you mean lowercasing them at generation time? Or just having a `alias-js` like with the other PR?

File: docs/src/api/class-websocket.md [hunk 1]
```diff
@@ -7,19 +7,19 @@ The [WebSocket] class represents websocket connections in the page.
...
 
-## event: WebSocket.framesent
+## event: WebSocket.frameSent
 - type: <[Object]>
   - `payload` <[string]|[Buffer]> frame payload
 
 Fired when the websocket sends a frame.
 
-## event: WebSocket.socketerror
+## event: WebSocket.sockEterror
```
Comment (yury-s) [inline docs/src/api/class-websocket.md:22]: sockEterror -> socketError

File: utils/generate_types/index.js [hunk 2]
```diff
@@ -184,7 +184,8 @@ function createEventDescriptions(classDesc) {
   if (!hasUniqueEvents(classDesc))
     return [];
   const descriptions = [];
-  for (const [eventName, value] of classDesc.events) {
+  for (let [eventName, value] of classDesc.events) {
```
Comment (yury-s) [inline utils/generate_types/index.js:187]: Is there a corresponding change in playwirght.dev?

File: docs/src/multi-pages.md [hunk 3]
```diff
@@ -246,4 +246,4 @@ page.on("popup", handle_popup)
 ```
 
 ### API reference
-- [`event: Page.popup`]
\ No newline at end of file
+- [`event: Page.popUp`]
```
Comment (yury-s) [inline docs/src/multi-pages.md:249]: Let's keep it all lower case, it's used as single word usually.

**Review by yury-s — APPROVED** [review]

File: utils/generate_types/index.js (hunk 2 above)
Comment (avodovnik) [reply utils/generate_types/index.js:187]: There is now: https://github.com/microsoft/playwright.dev/pull/104 :-)

**Comment by avodovnik** [comment]
Linting will fail until this gets merged: https://github.com/microsoft/playwright/pull/5309

**Pull request merged**